src/components/molecules/NodeCard.tsx

```diff
@@ -113,6 +114,7 @@ const NodeCard = ({
         <div className="text-xl font-bold text-white w-full overflow-hidden overflow-ellipsis">
           {title || "Untitled Node"}
         </div>
+        {isCurrent && <ComponentLibraryList />}
```

Comment:
please construct the list based on the components in the Node's manifest, found in the manifest/manifestData state in the `useNodeReader` hook